{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/components/specbut/specbut.jinja

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
This is debug code right?

Reply:
yes, it has been removed in the headless PR